pages/dkp/kommander/2.1/release-notes/2.1.2/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.2 Release Notes
+title: Kommander 2.1.2 Release Notes
+menuWeight: 30
+excerpt: View release-specific information for Kommander 2.1.2
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.2 was released on June 1, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## Fixes and Improvements
+
+### Upgrade Konvoy 1.8.4 to 2.1.1 index out of range error (COPS-7183)
+
+When attempting to upgrade a Konvoy v1.8 deployment that used an existing VPC, the `prepare-to-adopt` command triggered an error. This problem has been corrected.
+
+### Flux proxy configuration results in failure to install applications (COPS-7236)
+
+A Flux configuration problem that resulted in Kommander applications not being able to be pulled and installed in environments with a HTTP[s] Proxy configured was resolved.
+
+<!-- vale Microsoft.HeadingColons = NO -->
+### kube-oidc-proxy error: certificate signed by unknown authority (COPS-7217)
+<!-- vale Microsoft.HeadingColons = YES -->
+
+A problem that prevented successful configuration of the `kube-oidc-proxy` component when using a custom domain and TLS certificate issued by Let's Encrypt was corrected.
+
+### Certificate objects updated but not reloading in Kommander pods (COPS-7212)
```

Comment:
Forgot to take this out. Added the workaround in the "Known issues" section, but since the fix was provided in 2.2.1 only, we can deleted this section here.